Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -242,6 +248,11 @@ PatchAppleXcpmCfgLock (
     return EFI_SUCCESS;
   }
 
+  if (Patcher == NULL) {
```

Comment:
Makes no sense for kernel patches. Kernel can never be null. Same for certain other patches below.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/6535f1433e0d62b9b524cbd0120bde443bfa46e4